app/navigation/Navigator.scala

```diff
@@ -31,6 +32,9 @@ class Navigator {
     case IdentificationPage                          => ua => routeIdentificationPageNavigation(ua, departureId, mode)
     case CountryPage                                 => ua => AddressPage.route(ua, departureId, mode)
     case MoreInformationPage                         => ua => locationOfGoodsNavigation(ua, departureId, mode)
+    case CustomsOfficeIdentifierPage                 => ua => AddUnLocodePage.route(ua, departureId, mode)
+    case AddContactYesNoPage                         => ua => addContactYesNoPageNavigation(ua, departureId, mode)
+    case PhoneNumberPage                             => ua => phoneNumberPageNavigation(ua, departureId, mode)
```

Comment:
Might be best to pull in my branch as the navigators have been separated and the rest of the location of goods nav should be in the LocationOfGoodsNavigator.